config.yaml

```diff
@@ -10,6 +10,7 @@ default:
  release_notes_repo_url: "https://github.com/open-power-host-os/open-power-host-os.github.io.git"
  release_notes_repo_branch: "master"
  mock_binary: "/usr/bin/mock"
+ mock_config: "./extras/centOS/7/mock/epel-7-ppc64le.cfg"
```

Comment:
As we talked personally, there should be one Mock config file should be defined per distro and distro version. I noticed that the previous code (incorrectly) always used the same Mock config file for any distro version, but at least it used one different Mock config file for each distro. This patch allows one Mock config for all builds, reducing build script flexibility. Pls use subattribs in the yaml for 'centOS' and '7' as we have in the packages metadata (yaml) file.